R/data.R

```diff
@@ -20,15 +20,16 @@ NULL
 #'
 #' A dataset containing Ammonia, Nitrate, and Nitrogen data from Utah:
 #'  
-#' TADAProfile <- TADAdataRetrieval(statecode = "UT",
-#' characteristicName = c("Ammonia", "Nitrate", "Nitrogen"), 
-#' startDate = "10-01-2020")
+# TADAProfile <- TADAdataRetrieval(statecode = "UT",
```

Comment:
Can you please update these in the Shiny data.R and data folder as well for consistency (in a new PR/branch on the Shiny repo)? https://github.com/USEPA/TADAShiny/tree/develop/data